backend/app/utils/events.py

```
@@ -0,0 +1,141 @@
+        for queue in self._connections[user_id][:]:  # 创建副本避免修改迭代中的列表
+            try:
+                await queue.put(formatted_data)
+            except Exception:
+                # 连接已断开，标记待移除
+                disconnected_queues.append(queue)
+
+        # 清理断开的连接
+        for queue in disconnected_queues:
+            await self.remove_connection(user_id, queue)
+
+    async def notify_content_status(
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)